Sources/CacheAdvance/CacheHeaderHandle.swift

```diff
@@ -74,11 +74,11 @@ final class CacheHeaderHandle {
         try handle.write(data: expectedHeader.data(for: .offsetInFileAtEndOfNewestMessage))
     }
 
-    /// Checks if the expected header version matches the persisted header version.
+    /// Throws if the expected header version does not match the persisted header version.
```

Comment:
```suggestion
    /// Checks that the file is in a valid state ....<insert more?>....
```
Assuming that we go with the a name like `checkFile()` I wonder if we can make this more affirmative now and then keep the failure details in the `- Throws:` section?